### PR TITLE
[WIP] Feat/169 - Testnet implemented on hardhat node

### PR DESCRIPTION
**Pull request by sifnoc**
Not fully implemented hardhat into zkopru.

- Update docker-compose files for using hardhat node instead of ganache-cli as testnet container
- Removed Postgres from docker and related parts

Related issue #169

### CODE REVIEW

**Comment by sifnoc** [comment]
docker-compose file description

- `docker-compose.dev-scratch.yml`: This docker-compose file no need a source testchain for fork. the 'testnet' container will deploy Zkopru contract by self.
- `docker-compose.dev.yml`: Fork a source testchain(`"https://zkopru-testnet.sifnoc.net"`) which is already deployed Zkopru contracts. The testchain mine block every 10 sec.
- `docker-compose.instant-block.yml`: Fork a source testchain(`"https://zkopru-testnet.sifnoc.net"`) which is already deployed Zkopru contracts. the testchain is set automine It will mine a block if has a transaction in tx-pool.
- `docker-compose.yml`: Fork a source testchain(`"https://zkopru-testnet.sifnoc.net"`) which is already deployed Zkopru contracts. Same as `docker-compose.dev.yml' but no Wallet Container.

**Comment by wanseob** [comment]
@sifnoc Thx Jin for the hard work and your first PR to the project :) Do u have any recommended testing sequence?

**Comment by sifnoc** [comment]
Nothing changed existing testing, yet. 

I removed `contracts-for-integration-test` container inside `dockerfiles/docker-compose.yml` because It is duplicated when run `docker-compose.dev-scratch.yml`

So If you asked about the 'testing sequence', I can say we should run both at least.

- run `dockerfiles/docker-compose.yml` 
- run `/docker-compose.dev-scratch.yml`

File: packages/contracts/hardhat.config-fork.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ */
+module.exports = {
+  solidity: "0.7.4",
+  networks: {
+    hardhat: {
+      chainId: 20200406,
+      blockGasLimit: 12000000,
+      forking: {
+        url: "https://zkopru-testnet.sifnoc.net",
+        blockNumber: 95 // Migration step 19 complete as 'testnet' network.
```
Comment (sifnoc) [inline packages/contracts/hardhat.config-fork.js:12]: <table>
<tr>
	<td>blockNumber
	<td>migration file
	<td>description
<tr>
	<td>63
	<td>18_setup
	<td>deployed Zkopru Contract
<tr>
	<td>64
	<td>18_setup
	<td>setChallengePeriod as 30
<tr>
	<td>65
	<td>18_setup
	<td>register account to BurnAuction contract
<tr>
	<td>93
	<td>19_migration_dest
	<td>Deployed Zkopru2 contract as migration dest
<tr>
	<td>94
	<td>19_migration_dest
	<td>setChallengePeriod as 30
<tr>
	<td>95
	<td>19_migration_dest
	<td>register account to BurnAuction contract
</table>

**Comment by sifnoc** [comment]
Should wait for fix this issue #205 before merge this PR

**Comment by wanseob** [comment]
@sifnoc would you please add [WIP] to the title or mark it as a draft?

**Comment by sifnoc** [comment]
It seems fix `eth_subscribtion` at https://github.com/nomiclabs/hardhat/releases/tag/hardhat-core-v2.6.1

**Comment by sifnoc** [comment]
This PR is going to move `stress-test` repo.

Create issue https://github.com/zkopru-network/stress-test/issues/8 for this PR.

**Pull request closed without merging**